tools/publish-aos.js

```diff
@@ -1,54 +1,170 @@
 const Arweave = require('arweave');
 const path = require('path');
 const fs = require('fs');
+const { TurboFactory } = require('@ardrive/turbo-sdk');
+const { createData, ArweaveSigner } = require('@dha-team/arbundles');
+
+const changelog = `
+# Changelog
+
+### Changed
+
+- Changed default landing page transaction ID.
```

Comment:
can this be injected at build time and we can pull from a CHANGELOG file since last released?

Reply:
will manually update changelogs